src/cephadm/cephadm.py

```diff
@@ -1005,9 +1005,14 @@ def __init__(self,
         self.image = image
 
         self.sock_dir = config_json.get('sock-dir', '/var/run/ceph/')
-        ipv4_addrs, ipv6_addrs = get_ip_addresses(get_hostname())
-        # use the first ipv4 (if any) otherwise use the first ipv6
-        addrs = next(iter(ipv4_addrs or ipv6_addrs), None)
+        addrs = None
+        ipv4_addrs, _ = get_ip_addresses(get_hostname())
+        if ipv4_addrs:
+            addrs = "0.0.0.0"
+        else:
+            addrs = "::"
+        logger = logging.getLogger("waah")
```

Comment:
what is "waah"?

Reply:
This is still a draft PR, I'm still debugging some issue, it's not ready for review yet.